### PR TITLE
cells_data: add clock information on ports.json

### PR DESCRIPTION
**Pull request by acomodi**
Signed-off-by: Alessandro Comodi <acomodi@antmicro.com>

It is useful to have also the information whether a port is a clock in the cells_data files

### CODE REVIEW

**Comment by acomodi** [comment]
[Zynq has failed](https://source.cloud.google.com/results/invocations/bb972f75-a652-417b-9a81-ef6d8b0224cc/targets), but all the other targets went green. I think this is ready for review. I am re-running once again CI

File: utils/update_resources.py [hunk 1]
```diff
@@ -39,30 +41,42 @@ def main():
...
+        # Skip parts which differ only in the speedgrade, as they have the same pins
+        fields = part.split("-")
+        common_part = fields[0]
+        if common_part in processed_parts:
+            information[part] = processed_parts[common_part]
+            continue
+
         print("Find pins for {}".format(part))
         env['XRAY_PART'] = part
+        _, tmp_file = tempfile.mkstemp()
```
Comment (dnltz) [inline utils/update_resources.py:55]: Maybe remove the tempfile later?

**Review by dnltz — APPROVED** [review]
Nice improvements.

The zynq failed because the hostcheck is a little bit too hard. https://github.com/SymbiFlow/prjxray/pull/1632

File: utils/update_resources.py (hunk 1 above)
Comment (acomodi) [reply utils/update_resources.py:55]: Yep, makes sense, done.

**Comment by acomodi** [comment]
> The zynq failed because the hostcheck is a little bit too hard. #1632

Ok, I have rebased on top of #1632 and https://github.com/SymbiFlow/prjxray/pull/1627

**Review by litghost — COMMENTED** [review]
This PR seems ok, but it shouldn't be merged until the dependent PR's are merged.

**Comment by acomodi** [comment]
> but it shouldn't be merged until the dependent PR's are merged.

Sure, added a DNM just in case

**Comment by litghost** [comment]
PR needs rebase

**Pull request merged**